parquet/src/main/java/org/apache/iceberg/parquet/ParquetValueReaders.java

```diff
@@ -584,9 +585,9 @@ protected void addPair(Map<K, V> map, K key, V value) {
     private K key = null;
     private V value = null;
 
-    public void set(K key, V value) {
-      this.key = key;
-      this.value = value;
+    public void set(K setKey, V setValue) {
```

Comment:
Nit: `newKey` and `newValue` to follow conventions.

Reply:
I've renamed all set to new. I don't think `set-` was the best name for it, but couldn't think of anything else.